pallets/asset/src/types.rs

```diff
@@ -38,6 +42,14 @@ pub struct AssetInputEntry<AssetDescription, AssetTypeOf, AssetTag, AssetMeta> {
 	pub asset_meta: AssetMeta,
 }
 
+#[derive(
```

Comment:
@amarts I will discard `VCAssetInputEntry` & `VCAssetInputEntryOf` in lib.rs as asset_qty will be taken as a input parameter directly.